src/System.Memory/ref/System.Memory.csproj

```diff
@@ -14,7 +14,7 @@
     <Compile Include="System.Memory.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == 'netstandard1.0'">
-    <Reference Include="System.Runtime" />
+    <ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj" />
```

Comment:
We don't want to do this. The reason why your build is failing, is most likely because you are not building from root, which is who will have previously built this contract. Make sure you call `build.cmd` from the root at least once first so that you can build individual projects. Also, the specific targetGroup this will affect is netstandard1.0, so if you want to build that, you can also just call from the root: `build.cmd -allConfigurations` which will build the netstandard1.0 asset for you.